src/dataflow/source/kafka.rs

```diff
@@ -680,21 +726,8 @@ where
                                     (out, Some(Into::<KafkaOffset>::into(offset).offset)),
                                 ));
 
-                                KAFKA_PARTITION_OFFSET_INGESTED
-                                    .with_label_values(&[
-                                        &dp_info.topic_name,
-                                        &dp_info.source_id,
-                                        &partition.to_string(),
-                                    ])
-                                    .set(offset.offset);
-
-                                KAFKA_PARTITION_MESSAGE_INGESTED
-                                    .with_label_values(&[
-                                        &dp_info.topic_name,
-                                        &dp_info.source_id,
-                                        &partition.to_string(),
-                                    ])
-                                    .inc_by(1);
+                                partition_metrics.offset_ingested.set(offset.offset);
+                                partition_metrics.messages_ingested.inc_by(1);
```

Comment:
according to bwm we can just do `inc`

Reply:
👍 I'll do that in a separate PR to avoid eating another CI run